components/pkg-export-kubernetes/plan.sh

```diff
@@ -37,23 +37,24 @@ _common_prepare() {
 do_prepare() {
   _common_prepare
 
-  export rustc_target="x86_64-unknown-linux-musl"
```

Comment:
We can update this entire section to look a lot more like `./components/sup/plan.sh` as they are both dynamically linked against glibc. As a result, most of the linking here won't be needed.